test/extended/oauth/requestheaders.go

```diff
@@ -211,16 +211,20 @@ var _ = g.Describe("[Serial] [sig-auth][Feature:OAuthServer] [RequestHeaders] [I
 				respDump, err := httputil.DumpResponse(resp, true)
 				o.Expect(err).NotTo(o.HaveOccurred())
 				if len(tc.expectedError) == 0 && resp.StatusCode != 200 && resp.StatusCode != 302 {
+					gatherPostMortem(oc)
```

Comment:
Don't ginkgo or gomega provide a better mechanism to perform actions upon a failure?

Reply:
Could you be more explicit? You mean adding it to some `afterEach` function or do you mean that there exists already a `gatherPostMortem` in our test suites?